.circleci/config.yml

```diff
@@ -930,7 +930,7 @@ commands:
 
       - run:
           name: Build main image
-          command: make docker-build-main-image
+          command: make docker-build-main-image ROX_PRODUCT_BRANDING="<< parameters.branding >>"
```

Comment:
> Hey, I found also the following variable `REACT_APP_ROX_PRODUCT_BRANDING` - could you double-check whether its (default) value is in line with the `ROX_PRODUCT_BRANDING`?

@vikin91 As far as I can tell, we have two separate build paths in `config.yml`.  
For the `-stackrox` ones, we don't set `REACT_APP_ROX_PRODUCT_BRANDING`, which should result in the UI falling back to its default, which is `stackrox` (https://github.com/stackrox/stackrox/blob/d653406651df4331a714839ec2c0a23a93425c64/ui/apps/platform/src/constants/productBranding.ts#L44). Although I think it might be better to explicitly set this for the `-stackrox` UI build as we did for the `-rhacs` build. WDYT?

Reply:
I support setting values explicitly 👍  But maybe we should check `REACT_APP_ROX_PRODUCT_BRANDING` in a separate PR? We are not touching it here and I mentioned it here just to double check that all aligns with out expectations.

Reply:
True. AFAIK, we're not impacting/changing the default behaviour right now, so let's move on with this PR.
@jschnath could you create a follow up item so we don't forget to change this? Thanks 🥇